contracts/ReaperAutoCompoundXBoo.sol

```diff
@@ -338,12 +352,9 @@ contract ReaperAutoCompoundXBoov2 is ReaperBaseStrategyv3, IERC721ReceiverUpgrad
                 address(this),
                 block.timestamp
             );
-            uint256 USDCBal = IERC20Upgradeable(USDC).balanceOf(address(this));
-            callFeeToUser = (USDCBal * callFee) / PERCENT_DIVISOR;
-            uint256 treasuryFeeToVault = (USDCBal * treasuryFee) / PERCENT_DIVISOR;
+            feeCharged = IERC20Upgradeable(USDC).balanceOf(address(this));
```

Comment:
should we do a before/after balance of USDC just to be sure? thinking of a case where we start depositing into a new pool that gives out USDC as reward but forget to add a swap path for USDC.

Reply:
It is probably a good practice to do so.